### DISCUSSION TITLE
Custom difficulties of ranked mapsets cause unobserved error when playing.

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#19241] [Q&A] [answered]
Version 2022.719.0
I've made many practice difficulties to help prepare for a tournament. When updating to the newest version, playing these difficulties causes an unobserved error to occur every attempt. This didn't happen in 2022.716.1.
Exported mapset: [Houshou Marine - Ahoy!! Warera Houshou Kaizokudan* (X a v y).osz.zip](https://github.com/ppy/osu/files/9144257/Houshou.Marine.-.Ahoy.Warera.Houshou.Kaizokudan.X.a.v.y.osz.zip) | [runtime.log](https://github.com/ppy/osu/files/9144261/runtime.log) | [database.log](https://github.com/ppy/osu/files/9144265/database.log) | [network.log](https://github.com/ppy/osu/files/9144266/network.log)

From runtime.log:
```
[error]: An unobserved error has occurred.
[error]: Microsoft.AspNetCore.SignalR.HubException: An unexpected error occurred invoking 'BeginPlaySession' on the server.
[error]: at Microsoft.AspNetCore.SignalR.Client.HubConnection.InvokeCoreAsyncCore(String methodName, Type returnType, Object[] args, CancellationToken cancellationToken)
[error]: at Microsoft.AspNetCore.SignalR.Client.HubConnection.InvokeCoreAsync(String methodName, Type returnType, Object[] args, CancellationToken cancellationToken)
[error]: at osu.Game.Online.Spectator.OnlineSpectatorClient.BeginPlayingInternal(SpectatorState state) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Game/Online/Spectator/OnlineSpectatorClient.cs:line 59
```
I can record video of the error if it helps to diagnose.

**Comment by frenzibyte** [comment, marked as answer]
Has been fixed in https://github.com/ppy/osu/pull/19227, you can ignore the notifications for now.
